WcaOnRails/app/assets/stylesheets/avatars.scss

```diff
@@ -37,3 +37,16 @@
     background: rgba(0, 0, 0, 0.5);
   }
 }
+
+#upload-avatar-form{
```

Comment:
Nitpick: `#upload-avatar-form {` (having a space before braces) might be better to match other rules of the file.